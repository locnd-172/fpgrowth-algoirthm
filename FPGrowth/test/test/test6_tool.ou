22976:24
22748:24
22333:24
22432:25
22385:25
21987:25
21987 POST:24
23199:26
22661:26
22437:26
22437 POST:25
22435:26
22383:26
23256:27
23256 23254:25
23204:27
22659:27
22631:27
22367:27
21915:27
20749:27
20749 POST:25
22895:28
22895 POST:25
22620:28
22303:28
22303 POST:24
22029:28
20682:28
23254:29
23203:29
23203 POST:27
21936:29
21936 POST:27
20719:29
20719 POST:26
21121:30
21121 POST:25
20712:30
20712 POST:30
22555:32
22555 POST:25
22138:32
22090:32
22090 POST:28
23209:33
23209 POST:28
22662:34
22139:34
22727:37
22727 22728:29
22727 22728 POST:26
22727 22726:31
22727 22726 22728:25
22727 22726 POST:28
22727 POST:34
22027:37
22027 POST:35
20724:38
20724 POST:32
85099B:39
85099B POST:33
22726:39
22726 22728:29
22726 22728 POST:24
22726 POST:33
22728:40
22728 POST:35
22630:40
22630 22629:28
22630 POST:29
21212:40
21212 POST:33
22492:41
22492 POST:32
22328:44
22328 22326:26
22328 22326 POST:24
22328 POST:38
20726:47
20726 22382:26
20726 POST:40
22382:48
22382 20725:26
22382 23206:27
22382 POST:36
21559:48
21559 22352:25
21559 POST:45
22629:50
22629 POST:39
21094:50
21094 21080:40
21094 21080 POST:33
21094 21080 21086:39
21094 21080 21086 POST:32
21094 POST:42
21094 21086:48
21094 21086 POST:40
23206:51
23206 20725:27
23206 20725 POST:25
23206 POST:42
21080:52
21080 21086:40
21080 21086 POST:33
21080 POST:43
22423:54
22423 POST:42
21086:54
21086 POST:46
22551:55
22551 22556:36
22551 22556 22554:28
22551 22556 POST:29
22551 22554:42
22551 22554 POST:33
22551 POST:45
20750:55
20750 POST:45
22352:57
22352 POST:45
20725:61
20725 POST:48
22326:65
22326 POST:58
22556:68
22556 22554:41
22556 22554 POST:33
22556 POST:58
22554:68
22554 POST:54
21731:72
21731 POST:62
23084:75
23084 POST:65
POST:311
