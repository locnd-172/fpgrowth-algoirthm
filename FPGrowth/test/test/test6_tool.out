20682:28
20712:30
20719:29
20724:38
20725:61
20726:47
20749:27
20750:55
21080:52
21086:54
21094:50
21121:30
21212:40
21559:48
21731:72
21915:27
21936:29
21987:25
22027:37
22029:28
22090:32
22138:32
22139:34
22303:28
22326:65
22328:44
22333:24
22352:57
22367:27
22382:48
22383:26
22385:25
22423:54
22432:25
22435:26
22437:26
22492:41
22551:55
22554:68
22555:32
22556:68
22620:28
22629:50
22630:40
22631:27
22659:27
22661:26
22662:34
22726:39
22727:37
22728:40
22748:24
22895:28
22976:24
23084:75
23199:26
23203:29
23204:27
23206:51
23209:33
23254:29
23256:27
POST:311
85099B:39
20712 POST:30
20719 POST:26
20724 POST:32
20725 POST:48
20726 POST:40
20749 POST:25
20750 POST:45
21080 POST:43
21086 POST:46
21094 POST:42
21121 POST:25
21212 POST:33
21559 POST:45
21731 POST:62
21936 POST:27
21987 POST:24
22027 POST:35
22090 POST:28
22303 POST:24
22326 POST:58
22328 POST:38
22352 POST:45
22382 POST:36
22423 POST:42
22437 POST:25
22492 POST:32
22551 POST:45
22554 POST:54
22555 POST:25
22556 POST:58
22629 POST:39
22630 POST:29
22726 POST:33
22727 POST:34
22728 POST:35
22895 POST:25
23084 POST:65
23203 POST:27
23206 POST:42
23209 POST:28
20725 22382:26
20725 23206:27
20726 22382:26
21080 21086:40
21080 21094:40
21086 21094:48
21559 22352:25
22326 22328:26
22382 23206:27
22551 22554:42
22551 22556:36
22554 22556:41
22629 22630:28
22726 22727:31
22726 22728:29
22727 22728:29
23254 23256:25
85099B POST:33
20725 23206 POST:25
21080 21086 POST:33
21080 21094 POST:33
21086 21094 POST:40
22326 22328 POST:24
22551 22554 POST:33
22551 22556 POST:29
22554 22556 POST:33
22726 22727 POST:28
22726 22728 POST:24
22727 22728 POST:26
21080 21086 21094:39
22551 22554 22556:28
22726 22727 22728:25
21080 21086 21094 POST:32
